devops/scripts/check-commits.sh

```diff
@@ -9,7 +9,8 @@ set -euo pipefail
 . devops/scripts/commit-type.sh
 
 # List commits between release-pointer and HEAD, in reverse
-prcommits=$(git log $(git merge-base default HEAD)..HEAD --pretty=format:"%h")
+merge_base=$(git merge-base default HEAD)
+prcommits=$(git log "${merge_base}"..HEAD --pretty=format:"%h")
 
```

Comment:
```suggestion
merge_base=""
merge_base=$(git merge-base default HEAD)
prcommits=""
prcommits=$(git log "${merge_base}"..HEAD --pretty=format:"%h")
```
Mostly calling out just because I would have expected this to have been called out by the linter, but it wasn't. 🤷🏻  Ususally doesn't like if you define a variable at the same time you assign its subshell value as that can mask errors in the subshell.